styleguide/source/_patterns/01-molecules/06-components/15-table/_table.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file accordion.scss
+ *
+ * Copyright 2017 Palantir.net, Inc.
+ */
+
+.table {
+  @include grid__unit--cols(12);
+
+  @include breakpoint($bp-small) {
+    @include grid__unit--cols(10);
+    padding: 20px 0;
+  }
+  
+  @include breakpoint($bp-med) {
+    @include grid__unit--cols(6);
+  }
+
+  border-top: 1px solid $rule;
```

Comment:
In _global-classes.scss we have a  `.rule_horizontal {
  border-top: 1px $gray-8 solid;
}` class set up, so it would be ideal to combine these with a mixin, or note the reasons why we need both with inline comments.